vnext/codegen/NativeAlertManagerSpec.g.h

```diff
@@ -14,8 +14,27 @@
 namespace Microsoft::ReactNativeSpecs {
 
 struct AlertManagerSpec : winrt::Microsoft::ReactNative::TurboModuleSpec {
+  struct Args {
+      REACT_FIELD(title)
```

Comment:
>REACT_FIELD [](http://example.com/codeflow?start=6&length=11)

This attribute does not work without the REACT_STRUCT in front of the struct above.

Reply:
I see that the registration is done below instead. I wonder why it did not work in-place.

Reply:
It could because:

> First, the argument-dependent lookup is not considered if the lookup set produced by usual unqualified lookup contains any of the following:
>
> 1) a declaration of a class member
> 2) a declaration of a function at block scope (that's not a using-declaration)
> 3) any declaration that is not a function or a function template (e.g. a function object or another variable whose name conflicts with the name of the function that's being looked up)

https://en.cppreference.com/w/cpp/language/adl